### DISCUSSION TITLE
How to deploy Flowise AI?

### DISCUSSION THREAD

**Discussion by BorysPikalov** [FlowiseAI/Flowise#210] [Q&A]
I'm a complete nooby using Github for the first time, and I really want to play around with Flowise and the no-code space. Thus, it's not clear to me how to execute the deployment instruction, i.e., where do I write and run "npm install -g flowise" and "npx flowise start". 

Any advice would be super appreciated!

**Comment by Canabiense** [comment]
you need to use the 'command prompt' application. you can search for it on Windows tab. 

However, i had issues with this installation. there is some issue with the folders downloaded. I ended up using Render (for free) to host it and play with it there.
